### PR TITLE
finished

### PR DESCRIPTION
**Pull request by barantarasNEW**
https://barantarasNEW.github.io/react_tabs-with-router/

### CODE REVIEW

File: src/Components/Nav/Nav.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+const selectedNav = (isActive: boolean) => classNames(
```
Comment (OlEzhA0) [inline src/Components/Nav/Nav.tsx:4]: not blocker: 
some trick
you can accept here 
`const selectedNav = ({ isActive }: { isActive: boolean }) => classNames(`

and use like

```
  <NavLink
    to="/"
    className={selectedNav}
  >
    Home
  </NavLink>
```

File: src/Components/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+};
+
+const Tabs: React.FC<Props> = ({ tabs }) => {
+  const { tabId = 0 } = useParams();
+  const foundTab = tabs.find(tab => tab.id === tabId) || null;
```
Comment (OlEzhA0) [inline src/Components/Tabs/Tabs.tsx:11]: add useMemo

File: src/Components/Tabs/Tabs.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import { NavLink, useParams } from 'react-router-dom';
+import classNames from 'classnames';
```
Comment (OlEzhA0) [inline src/Components/Tabs/Tabs.tsx:2]: in projects this library is usually called like `cn` or `cx`

File: src/Components/Tabs/Tabs.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+              <NavLink to={`../${tab.id}`}>
+                {tab.title}
+              </NavLink>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {foundTab ? foundTab.content : 'Please select a tab'}
```
Comment (OlEzhA0) [inline src/Components/Tabs/Tabs.tsx:34]:
```suggestion
        {foundTab?.content || 'Please select a tab'}
```

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
hey! just few imporvements

File: src/Components/Tabs/Tabs.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+const Tabs: React.FC<Props> = ({ tabs }) => {
+  const { tabId = 0 } = useParams();
+  const foundTab = useMemo(() => {
+    return tabs.find(tab => tab.id === tabId) || null;
+  }, [tabId]);
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
```
Comment (mykhalenych) [inline src/Components/Tabs/Tabs.tsx:18]: you already have 3 h1 but semantically right use only 1

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**